### PR TITLE
Make runQuery accept a DocumentNode

### PR DESCRIPTION
**Pull request by ramirotw**
Right now the runQuery method only accepts a string and doesn't accept a DocumentNode when the query is parsed through Apollo's gql function, this forces the caller to cast the query as in https://github.com/cowprotocol/explorer/pull/93/files#diff-02bebd690fca011b032ac0e74e3286c0d751bc1f454eb09492f6aaac0c2d0ed3R50

### CODE REVIEW

**Review by matextrem — APPROVED** [review]

File: src/api/cow/index.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ import {
 import { CowError, logPrefix, objectToQueryString } from '../../utils/common'
 import { Context } from '../../utils/context'
 
+export { OperatorError, QuoteError }
+
```
Comment (alfetopito) [inline src/api/cow/index.ts:30]: This doesn't seem related to the change in question.
What is this for/from?

**Review by alfetopito — APPROVED** [review]

File: src/api/cow/index.ts (hunk 1 above)
Comment (ramirotw) [reply src/api/cow/index.ts:30]: did some tests and forgot to remove it, thanks for catching it up.
